2019-03-25 12:28:05,403:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 12:28:06,072:WARNING:Not Found: /favicon.ico
2019-03-25 12:28:08,319:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:28:08,319:ERROR:Internal Server Error: /newnumbertype
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\nFilter\controller\views.py", line 69, in getprepix
    if NumberType.objects.filter(id=numbertype).exists():
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 670, in exists
    return self.query.has_results(using=self.db)
  File "C:\Python27\lib\site-packages\django\db\models\sql\query.py", line 517, in has_results
    return compiler.has_results()
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 853, in has_results
    return bool(self.execute_sql(SINGLE))
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 894, in execute_sql
    raise original_exception
OperationalError: no such table: numbertype
2019-03-25 12:38:00,763:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 12:38:02,627:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:38:03,151:WARNING:Not Found: /favicon.ico
2019-03-25 12:38:05,460:WARNING:Not Found: /numberPrice
2019-03-25 12:38:09,556:WARNING:Not Found: /getprefix
2019-03-25 12:42:05,013:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:42:11,224:WARNING:Not Found: /numberPrice
2019-03-25 12:43:17,184:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:43:18,721:INFO:ДУГААРЫН ҮНЭ
2019-03-25 12:43:20,206:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:43:25,595:INFO:ДУГААРЫН ҮНЭ
2019-03-25 12:43:26,447:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:43:31,588:INFO:ДУГААРЫН ҮНЭ
2019-03-25 12:43:32,868:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:43:40,756:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 12:43:42,608:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:43:52,206:WARNING:Not Found: /selectednumber
2019-03-25 12:44:20,927:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 12:44:24,157:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:44:41,265:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:44:52,497:INFO:ДУГААРЫН ҮНЭ
2019-03-25 12:44:53,377:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:44:59,401:INFO:ДУГААРЫН ҮНЭ
2019-03-25 12:45:00,267:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 12:45:21,486:WARNING:Not Found: /selectednumber
2019-03-25 12:55:24,970:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 13:55:25,505:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 14:48:59,776:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 14:50:53,749:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 14:50:55,420:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 14:51:53,551:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 14:51:55,181:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 14:52:37,668:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 14:52:39,180:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 15:01:28,690:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 15:02:02,622:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 15:02:39,479:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 15:02:54,279:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 15:02:58,930:WARNING:Not Found: /favicon.ico
2019-03-25 15:03:04,082:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 15:03:52,706:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 15:05:06,862:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 15:05:14,891:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 15:05:14,892:ERROR:Internal Server Error: /registr
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\nFilter\controller\views.py", line 90, in registrDugaar
    turul = request.GET['turul']
  File "C:\Python27\lib\site-packages\django\utils\datastructures.py", line 85, in __getitem__
    raise MultiValueDictKeyError(repr(key))
MultiValueDictKeyError: "'turul'"
2019-03-25 15:05:41,723:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 15:05:41,723:ERROR:Internal Server Error: /registr
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\nFilter\controller\views.py", line 92, in registrDugaar
    hel = request.GET['hel']
  File "C:\Python27\lib\site-packages\django\utils\datastructures.py", line 85, in __getitem__
    raise MultiValueDictKeyError(repr(key))
MultiValueDictKeyError: "'hel'"
2019-03-25 15:09:47,115:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 15:09:47,124:INFO:Success
2019-03-25 15:11:03,259:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 15:11:03,259:ERROR:Internal Server Error: /registr
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\nFilter\controller\views.py", line 100, in registrDugaar
    soc.connect(('10.10.10.173', 80881))
  File "C:\Python27\lib\socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
OverflowError: getsockaddrarg: port must be 0-65535.
2019-03-25 15:26:41,252:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 15:26:41,269:INFO:Success
2019-03-25 15:26:42,615:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 15:36:42,789:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:02:40,589:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:06:06,717:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:06:12,638:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 16:06:23,255:WARNING:Not Found: /zahialga
2019-03-25 16:09:12,165:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:09:13,309:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:09:18,615:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 16:09:27,407:INFO:ДУГААРЫН ҮНЭ
2019-03-25 16:09:27,417:INFO:Success
2019-03-25 16:09:31,243:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:09:34,267:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:09:38,953:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:09:40,332:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:09:44,908:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 16:09:46,298:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:10:17,757:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 16:10:20,596:INFO:ДУГААРЫН ҮНЭ
2019-03-25 16:10:23,608:INFO:Success
2019-03-25 16:10:29,275:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:10:31,747:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:18:23,934:INFO:ДУГААРЫН ҮНЭ
2019-03-25 16:18:25,155:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:18:32,469:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:18:47,825:INFO:РЕГИСТЕРИЙН ДУГААР
2019-03-25 16:18:56,282:INFO:ДУГААРАА СОНГОЖ БУЙ ХЭСЭГ
2019-03-25 16:28:25,354:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 16:28:56,411:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 17:28:25,539:INFO:----------------------------------------------------------------------------------------------------
2019-03-25 17:28:57,233:INFO:----------------------------------------------------------------------------------------------------
